SPECS/python-retrying/python-retrying.spec

```diff
@@ -0,0 +1,142 @@
+# Created by pyp2rpm-1.1.0b
+%global pypi_name retrying
+
+Name:           python-%{pypi_name}
+Version:        1.3.3
+Release:        6%{?dist}
+Summary:        General-purpose retrying library in Python
+License:        ASL 2.0
+Vendor:         Microsoft Corporation
+Distribution:   Vendor
+URL:            https://github.com/rholder/retrying
+Source0:        https://pypi.python.org/packages/source/r/%{pypi_name}/%{pypi_name}-%{version}.tar.gz#/%{pypi_name}-%{version}.tar.gz
+BuildArch:      noarch
+
+%global _description\
+Retrying is an Apache 2.0 licensed general-purpose retrying library,\
+written in Python, to simplify the task of adding retry behavior to\
+just about anything.\
+
+%description %_description
+
+%package -n python3-%{pypi_name}
+Summary:        General-purpose retrying library in Python
+BuildRequires:  python3-devel
+BuildRequires:  python3-setuptools
+Requires:       python3-six
+
+%description -n python3-%{pypi_name}
+Retrying is an Apache 2.0 licensed general-purpose retrying library,
+written in Python, to simplify the task of adding retry behavior to
+just about anything.
```

Comment:
This should use the %_description macro that's defined above, instead of repeating the same text.

Reply:
Done.